src/main/java/ca/team2706/frc/robot/OI.java

```diff
@@ -65,9 +83,45 @@ public OI(Joystick driverStick, Joystick controlStick) {
 
         driveCommand = new CurvatureDriveWithJoystick(driverStick, Config.CURVATURE_DRIVE_FORWARD, true,
                 Config.CURVATURE_CURVE_SPEED, false, Config.SLOW_MODE);
+        liftCommand = new HoldLift();
 
         // Set subsystem default commands
         DriveBase.getInstance().setDefaultCommand(driveCommand);
+        Lift.getInstance().setDefaultCommand(liftCommand);
+
+        // Operator controls.
+        new FluidButton(controlStick, Config.INTAKE_BACKWARD_BINDING, 0.05)
+                .whenHeld(new RunIntakeOnJoystick(controlStick, Config.INTAKE_BACKWARD_BINDING, false));
+        new FluidButton(controlStick, Config.INTAKE_FORWARD_BINDING, 0.05)
+                .whenHeld(new RunIntakeOnJoystick(controlStick, Config.INTAKE_FORWARD_BINDING, true));
+        new FluidButton(controlStick, Config.MOVE_LIFT_BINDING, 0.05)
+                .whenHeld(new MoveLiftJoystickVelocity(controlStick, Config.MOVE_LIFT_BINDING));
+        new FluidButton(controlStick, Config.LIFT_ARMS_BINDING)
+                .whenPressed(new RaiseArmsSafely());
+        new FluidButton(controlStick, Config.LOWER_ARMS_BINDING)
+                .whenPressed(new LowerArmsSafely());
+        new FluidButton(controlStick, Config.OVERRIDE_LIFT_DOWN_BINDING)
+                .whenHeld(new MoveLiftOnOverride(false));
+        new FluidButton(controlStick, Config.OVERRIDE_LIFT_UP_BINDING)
+                .whenHeld(new MoveLiftOnOverride(true));
+        new FluidButton(controlStick, Config.LIFT_FIRST_SETPOINT_BINDING)
+                .whenHeld(new MoveLiftToSetpoint(0));
+        new FluidButton(controlStick, Config.LIFT_SECOND_SETPOINT_BINDING)
+                .whenHeld(new MoveLiftToSetpoint(1));
+        new FluidButton(controlStick, Config.LIFT_THIRD_SETPOINT_BINDING)
+                .whenHeld(new MoveLiftToSetpoint(2));
+        new FluidButton(controlStick, Config.LIFT_FOURTH_SETPOINT_BINDING)
+                .whenHeld(new MoveLiftToSetpoint(3));
+        new FluidButton(controlStick, Config.MANUAL_PISTON_BINDING)
+                .whenPressed(new MovePlunger());
+        FluidButton button = new FluidButton(controlStick, Config.EJECT_BINDING);
+        LiftPosition position = new LiftPosition();
+        button.whenHeld(new EjectConditional(position));
+        button.whenReleased(new AfterEjectConditional(position::getPosition));
+        new FluidButton(controlStick, Config.AUTO_INTAKE_CARGO_BINDING)
+                .whenHeld(new AutoIntakeCargo());
+        new FluidButton(controlStick, Config.TOGGLE_RING_LIGHT_BINDING)
+                .whenPressed(new ToggleRingLight());
 
         // Operator controls
         new FluidButton(driverStick, Config.DRIVER_ASSIST_VISION_CARGO_AND_LOADING_BINDING)
```

Comment:
Double check whether these should be `whenPressed()` or `whenHeld()`.

Reply:
Checked, changed a couple of `whenHeld()` (the snap ones) but the rest remained `whenPressed()`.